src/containers/my-resources/lessons-container/LessonsContainer.constants.tsx

```diff
@@ -0,0 +1,84 @@
+import ListAltIcon from '@mui/icons-material/ListAlt'
+import { Box, Typography } from '@mui/material'
+import { defaultResponses } from '~/constants'
+
+import { styles } from '~/containers/my-resources/lessons-container/LessonsContainer.styles'
+import { authRoutes } from '~/router/constants/authRoutes'
+import {
+  AdditionalPropsInterface,
+  Lesson,
+  Offer,
+  RemoveColumnRules,
+  SortEnum,
+  TableColumn
+} from '~/types'
+import { createUrlPath, getFormattedDate } from '~/utils/helper-functions'
+
+export const columns: TableColumn<Lesson>[] = [
+  {
+    label: 'myResourcesPage.lessons.title',
+    field: 'title',
+    calculatedCellValue: (
+      item: Lesson,
+      { navigate }: AdditionalPropsInterface
+    ) => (
+      <Box
+        onClick={() =>
+          navigate(
+            createUrlPath(authRoutes.myResources.myLesson.path, item._id)
+          )
+        }
+        sx={styles.lessonTitleContainer}
+      >
+        <ListAltIcon sx={styles.lessonIcon} />
+        <Typography sx={styles.lessonTitle}>{item.title}</Typography>
+      </Box>
+    )
+  },
+  {
+    label: 'myResourcesPage.lessons.attachments',
+    calculatedCellValue: (item: Lesson, { t }: AdditionalPropsInterface) => {
+      const attachmentsQty = item.attachments.length
+
+      return (
+        <Typography sx={styles.attachmentsTitle}>
+          {attachmentsQty}{' '}
+          {t(
+            `myResourcesPage.lessons.attachment${
+              attachmentsQty === 1 ? '' : 's'
+            }Qty`
+          )}
+        </Typography>
+      )
+    }
+  },
+  {
+    label: 'myResourcesPage.lessons.lastUpdates',
+    field: 'updatedAt',
+    calculatedCellValue: (item: Lesson) => (
+      <Typography sx={styles.dateTitle}>
+        {getFormattedDate(item.updatedAt)}
+      </Typography>
+    )
+  }
+]
+
+export const removeColumnRules: RemoveColumnRules<Offer> = {
+  tablet: [
+    'myOffersPage.tableHeaders.updated',
+    'myOffersPage.tableHeaders.status'
+  ]
+}
+
+export const defaultResponse = {
```

Comment:
It is redundant, you can use default responses.itemsWithCount 
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/63456632/56202cb2-5be6-41ac-8ae2-4a5e901dcd2e)